### ISSUE TITLE
notice of file downloads via IP capture and email to admin 

### ISSUE THREAD

**Issue by chuckingit** [owncloud/activity#350]
Hi - not sure where to put this suggestion for improving the Activity app but for me the most important activity that i'd want to track is not currently available with this app and that is an alert of when files are downloaded from shares ... ideally as admin i would love to get email (or peruse a log file) with the IP address along with date-time stamp of which files are being downloaded ... this would help in the event a file was being overly shared - meaning if i had private link that i sent to one person and then saw ten download notices from ten different IP addresses for that share, i would be alerted to the fact that the share link got out into the wild and therefore shut it down to block further shares ... perhaps this IP tracking downloads feature / functionality belongs in its own app / plugin ..??.. suffice it to say, i would love to know when and from where files are being accessed ... the activity within ownCloud of new users created, shares created, etc is not nearly as important IMHO as knowing activity like how often files are being downloaded / accessed and from what IPs ... cordially, chuck scott

**Comment by enoch85** [comment]
+1 for this! The IP/email part is a very cool idea. What about showing the IP as a tooltip in the activity feed as well?

**Comment by nickvergessen** [comment]
yeah of course it sounds cool, I also think its the way to go, hopefully we can add that for a future version

**Comment by enoch85** [comment]
This is still valid

**Comment by thecobe** [comment]
for me this would be a great feature

**Comment by wfunderberg** [comment]
Agreed, this is a much needed feature.

**Comment by enoch85** [comment]
@nickvergessen Something for Nextcloud?

**Comment by mutipg** [comment]
For me this would be a great feature too!

**Comment by jnihil** [comment]
I would also love this

**Comment by rudebwoyslops** [comment]
Is this still being considered as an option. I would love to have the ip address where the user downloaded from be included in the emails we get from activity. That would help when some one says that they didnt download the files
